packages/module/src/CodeModal/CodeModal.tsx

```diff
@@ -94,16 +97,17 @@ export const CodeModal: React.FunctionComponent<CodeModalProps> = ({
       ouiaId="CodeModal"
       aria-labelledby="code-modal-title"
       aria-describedby="code-modal"
-      className={`pf-chatbot__code-modal pf-chatbot__code-modal--${displayMode}`}
+      className={`pf-chatbot__code-modal ${isCompact ? 'pf-chatbot__code-modal-compact' : ''} pf-chatbot__code-modal--${displayMode}`}
```

Comment:
I suggest using a modifier class rather than this extra long class name for consistency with other PF components: `pf-m-compact`

You can see how that's called out in the HTML docs for table here: https://www.patternfly.org/components/table/html#compact-usage

Reply:
All set!